packages/manager/modules/netapp/src/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,3 @@
+{
+  "netapp_title": "NetApp"
```

Comment:
Not sure translation is needed if we keep "NetApp" for all but nitpicking

Reply:
Yes I was thinking the same, but I choose to stick with what is done for other modules, like sms or iplb